geemap/common.py

```diff
@@ -16167,3 +16167,160 @@ def is_on_aws():
         if item.endswith(".aws") or "ec2-user" in item:
             on_aws = True
     return on_aws
+
+
+def google_map_tiles(
+    map_type: str = None,
+    language: str = "en-Us",
+    region: str = "US",
+    api_key: Optional[str] = None,
+    **kwargs: Any,
+):
+    """
+    Generates Google Map tiles using the provided parameters. To get an API key and enable Map Tiles API, visit https://developers.google.com/maps/get-started#create-project.
+        You can set the API key using the environment variable `MAPS_API_KEY` or by passing it as an argument.
+
+    Args:
+        map_type (str, optional): The type of map to generate. Options are 'roadmap', 'satellite', 'terrain', 'hybrid', 'traffic', 'streetview'.
+            Defaults to None, which generates all map types, and returns a dictionary of TileProvider objects.
+        language (str, optional): An IETF language tag that specifies the language used to display information on the tiles, such as 'zh-Cn'. Defaults to 'en-Us'.
+        region (str, optional): A Common Locale Data Repository region identifier (two uppercase letters) that represents the physical location of the user. Defaults to 'US'.
+        api_key (str, optional): The API key to use for the Google Maps API. If not provided, it will try to get it from the environment or Colab user data. Defaults to None.
+        **kwargs: Additional parameters to pass to the map generation. For more info, visit https://developers.google.com/maps/documentation/tile/session_tokens#optional_fields
+
```

Comment:
If possible, try to line wrap the code comments at 80 characters to make it easier to review.

Reply:
I have formated the docstrings with the line length limit <80 characters. I have tried to set the docstring line limit in pre-commit, but it does not work. Will investigate later.